blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -242,10 +242,42 @@ function clone(json) {
 }
 
 export const actions = {
-    clearPipelinesData: () => ({ type: ACTION_TYPES.CLEAR_PIPELINES_DATA }),
+    clearPipelinesData(organizationName) {
+        return (dispatch) => dispatch({ type: ACTION_TYPES.CLEAR_PIPELINES_DATA, payload: organizationName });
```

Comment:
right. i did have it implemented, then changed it. i'll undo this. sorry for the leftover